src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,54 @@
+type Message = {
+    text: string;
+    type?: string;
+    action?: string;
+};
+
+type Button = {
+    text?: string;
+    tooltip?: string;
+    disabled?: boolean;
+    hidden?: boolean;
+    data: {
```

Comment:
```suggestion
    data?: {
```

According to the propTypes.